config/custom-environment-variables.yaml

```diff
@@ -180,6 +180,8 @@ plugins:
         blockedBySelf: PLUGIN_BLOCKEDBY_BLOCKED_BY_SELF
         # enable collapse feature or not
         collapse: PLUGIN_BLOCKEDBY_COLLAPSE
+        # default value for collapse annotation, if true, by default all builds will be collapsed and vice versa
+        defaultCollapseAnnotation: PLUGIN_BLOCKEDBY_DEFAULT_COLLAPSE_ANNOTATION
```

Comment:
this ENV is kinda confusing since `annotations` is more for specific pipeline, for cluster level can't we just name it like `PLUGIN_BLOCKEDBY_ENABLE_COLLAPSE`?

Reply:
this is the default value for `screwdriver.cd/collapseBuilds` and we have a feature flag called `PLUGIN_BLOCKEDBY_COLLAPSE` already. And that is for enabling the feature or not.

Reply:
maybe we don't need a feature flag to turn the whole thing off. we will turn the current flag into the default behavior